src/main/java/UseCase2.java

```diff
@@ -41,10 +52,19 @@ public static void main(String[]args)
                         select(customers.col("*")).
                         orderBy(customers.col("customer_id"));
                 result.show();
+                long count=result.count();
```

Comment:
While printing records try to put some limit in show method
try to avoid use of spark action like count() between code